tests/test_bme688/CMakeLists.txt

```diff
@@ -51,11 +51,13 @@ target_sources(${CMAKE_PROJECT_NAME} PRIVATE
 target_include_directories(${CMAKE_PROJECT_NAME} PRIVATE
     # Add user defined include paths here
     # Example: ${CMAKE_SOURCE_DIR}/../../Src/Sensors/Inc
+    ${CMAKE_CURRENT_SOURCE_DIR}/../../libs/sensors/Inc
 )
 
 # Add project symbols (macros)
 target_compile_definitions(${CMAKE_PROJECT_NAME} PRIVATE
     # Add user defined symbols for local scope to target executable
+    # DEBUG
```

Comment:
It seems like this may be superfluous, given the `CMAKE_BUILD_TYPE`, but I couldn't tell for sure yet.

Reply:
Ok, I figured out CubeMX handles this in `<target>/cmake/stm32cubemx/CMakeLists.txt`, with the following:
```cmake
target_compile_definitions(stm32cubemx INTERFACE 
	USE_HAL_DRIVER 
	STM32L476xx
    $<$<CONFIG:Debug>:DEBUG>
)
```